pkg/core/resources/store/options.go

```diff
@@ -161,6 +161,7 @@ type ListOptions struct {
 	FilterFunc   ListFilterFunc
 	NameContains string
 	Ordered      bool
+	ResourceKeys []core_model.ResourceKey
```

Comment:
I'd go with a `map[core_model.ResourceKey]struct{}` to use this as a set in other places.